.github/workflows/unit_test.yml

```diff
@@ -30,8 +31,17 @@ jobs:
       - name: Unit tests
         run: bundle exec fastlane unit
 
+      - name: Archive failure diffs
```

Comment:
We don't currently have any snapshot tests in this project, but I am inclined to leave this here as it doesn't break the build and we should add some.